spec/models/mixins/external_url_spec.rb

```diff
@@ -0,0 +1,42 @@
+describe ExternalUrlMixin do
+  let(:test_class) do
+    Class.new(ActiveRecord::Base) do
+      def self.name; 'TestClass'; end
+      self.table_name = 'vms'
+      include ExternalUrlMixin
+    end
+  end
+
+  describe '#external_url=' do
+    before do
+      User.current_user = FactoryBot.create(:user)
+    end
+
+    let(:test_instance) do
+      i = test_class.create
```

Comment:
I think something with `.tap` would be cleaner here.

```ruby
test_class.create.tap { |i| i.external_url = "https://www.example.com" }
```

Reply:
ok